torch/_inductor/codegen/triton_utils.py

```diff
@@ -99,4 +99,4 @@ def is_aligned(x: KernelArgType, alignment: int, include_tensor: bool) -> bool:
         for i, arg in enumerate(args)
         if is_aligned(arg, alignment=8, include_tensor=False)
     )
-    return instance_descriptor(divisible_by_16, (), (), divisible_by_8)
+    return create_instance_descriptor(divisible_by_16, (), divisible_by_8)
```

Comment:
This call site provide 3 arguments, but the function is defined to require 4 arguments. This should fail?

Reply:
Yes it was just a rebase typo as the scope was to maintain the original signature.